__tests__/test.js

```diff
@@ -92,6 +92,44 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+  describe("PATCH request", () => {
+    test("should accept an object in the form { inc_votes: newVote }, returning 200 and the updated article", () => {
+      const expectedArticle = {
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 110,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 10 })
+        .expect(200)
+        .then(({ body }) => {
+          const article = body;
+          expect(article).toMatchObject(expectedArticle);
+        });
+    });
+    test("should return 400 and a message of no new vote object if not passed a newVote object", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("no new vote object");
+        });
+    });
+    test('should return 404 and a message of article id not found if artcile id not found', () => {
+        return request(app)
+        .patch('/api/articles/9999')
+        .expect(404)
+        .then(({body}) => {
+            expect(body.msg).toBe('article id not found')
+        })
+    })
```

Comment:
Your tests should be:
- Status 400: E.g. PATCH /api/articles/notanumber
- Status 404: E.g. PATCH /api/articles/9999
- Status 400 - E.g. inc_votes value is not a number.

You're not sending anything for your sad path tests, have a look at the notes for post, this will give you an idea of how to set it up: https://notes.northcoders.com/courses/js-back-end/supertest#supertest-setup